code/__DEFINES/dcs/signals/signals_atom/signals_atom_attack.dm

```diff
@@ -0,0 +1,50 @@
+// All signals. Format:
```

Comment:
You're going to have to change all these comments from "all signals" to be more specific about what should and shouldn't be in here.